src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/ApplicationService.kt

```diff
@@ -165,7 +165,7 @@ class ApplicationService(
       this.data = data
     }
 
-    val savedApplication = applicationRepository.save(application)
+    val savedApplication = applicationRepository.save(removeXssCharacters(application))
```

Comment:
I'm wondering if it would be a bit better to be more specific about what we are doing here, by passing in just the `data` blob to this function before or at the same time as the `apply` above on line 164? 
I think passing the whole `application` in makes it read as if more fields than just the `data` field are being modified...

Reply:
I can look at this Harriet.  There aren't any other application fields we need to apply this sanitisation logic to are there?

Reply:
There's no other user-generated content except the `data` field currently so I don't think so

Reply:
Changed this now.  No need for a dedicated function since we only do this here.  https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1631/files